### ISSUE TITLE
Maclisp version of Eliza

### ISSUE THREAD

**Issue by larsbrinkhoff** [jeffshrager/elizagen.org#11]
Hello,

I have a version of ELIZA ported to Maclisp, dated 1977-09-05.  Unfortunately, it's just a compiled binary file.  Is this of any interest to you?

**Comment by jeffshrager** [comment]
Hmmm. Interesting. Is the any way to either run it, or extract the embedded code?

**Comment by larsbrinkhoff** [comment]
I may be able to demo this, and a few other ITS programs, at the Seattle VCF in March.

**Comment by jeffshrager** [comment]
OMG, you have an actual ITS system running?!

**Comment by larsbrinkhoff** [comment]
Yes.  ITS was actually first revived on an emulator in 1992 (two years after its demise) by former ITS hacker now Google engineer Ken Harrenstien.  Another emulator by DEC retiree Bob Supnik was used to bring ITS online on the Internet in 2001.  My contribution is to collect everything together in a single git repository with automated build scripts: http://github.com/PDP-10/its

Sorry to blather, but it's my favourite subject!

**Comment by larsbrinkhoff** [comment]
It's possible to read out some data with the FASLREAD package.  See https://github.com/PDP-10/its/issues/1081#issuecomment-406455739

CC @eswenson1

**Comment by larsbrinkhoff** [comment]
There is also a file called DOCTOR JONL2 which seems promising.  Note the dating on the script - 25 NOV 69.

https://github.com/PDP-10/its/blob/master/src/libdoc/doctor.jonl2#L300

**Comment by jeffshrager** [comment]
Hmmm. I wonder if I should try to give a talk at VCF on the ElizaGen project. Do you think that's the sort of thing folks might be interested in?

**Comment by dmberry** [comment]
Hi

People subscribed to this code might be interested in a book that has recently been published called: Hello, I’m ELIZA. Full reference below:

Baranovskaa, Marianna/Höltgen, Stefan (Hgg.): Hello, I’m Eliza. 50 Jahre Gespräche mit Computern. (Reihe: Computerarchäologie, Band 5). Bochum: Projektverlag 2018.

The contents page is copied below. 

Best

Davids


——



Vorwort der Reihenherausgeber ...................................................... 7 

0 Vorwort: TELL ME YOUR PROBLEM (Baranovska, Höltgen) ........... 9 

I ELIZAs Geschichte(n) ............................................................. 21

1  ELIZA – Ein Computerprogramm zur Untersuchung
der Kommunikation zwischen Mensch und Maschine
mithilfe natürlicher Sprache (Weizenbaum, van Treeck) ................... 23

2  Weizenbaum, Eliza and the End of Human Reason (Berry) ............... 53

II ELIZAs Sprachen ................................................................... 71

3  SLIP – Eine Sprache zwischen den Stühlen (Kantel) ....................... 73

4  ELIZA in BASIC (Shrager) ....................................................... 89

5  Computer/sprachen: ELIZA und BASIC – Urszenen des Homecomputings (und) künstlicher Intelligenz (Höltgen) ............... 97

III ELIZAs Folgen ..................................................................... 123

6  Die Maschine spricht Spannung (Baranovska) .............................. 125

7  From ELIZA to modern Chatbots and Dialog Systems – The Historical Development of Conversational So􏰀ware: Classification, Technology
and Application Domains (Le, Pinkwart, Masche) ......................... 143

8  Natürliche und künstliche Intelligenz –
50 Jahre nach ELIZA (Mainzer) ................................................. 161

9  Sprachspiele|Stimmsynthesen – Zur nachrichtentechnischen Genese
des auditiven Pendants von ELIZA (Borbach) ............................... 177

10  Die Tücke im System –
»Starke« KI im Science-Fiction-Film (Baum) ................................ 199

11  Sprechen/Versprechen – Sprachausgabe in Computerspielen
der frühen 1980er-Jahre (Keichel) .............................................. 211

12  The Human is the Problem – Scenes of Technology and Reason
in Media Performance (Leitner, Jucan, Pohle, Kipka) ...................... 225

Anhang .................................................................................. 241 Über die Autoren ................................................................. 243 Register .............................................................................. 251 Internetquellen .................................................................... 257

**Comment by larsbrinkhoff** [comment]
Yes, I think this is exactly the kind of thing people would be interested in!
